Makefile

```
@@ -202,3 +202,9 @@ JSONNETFMT_CMD := $(JSONNETFMT) -n 2 --max-blank-lines 2 --string-style s --comm
 .PHONY: jsonnet-fmt
 jsonnet-fmt: | $(JSONNETFMT)
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin echo ${JSONNET_SRC} | xargs -n 1 -- $(JSONNETFMT_CMD) -i
+
+.PHONY: generate-rules-spec
+generate-rules-spec:
+	@oapi-codegen -generate client -o rules/client.go -package rules rules/spec.yaml
```

We should definitely use Bingo to vendor and version oapi-gen as a tooling dependency of this project so that we can reproducibly regenerate code, so all devs are guaranteed to use the same oapi-gen version and CI can verify the generation
```suggestion
	$(OAPIGEN) -generate client -o rules/client.go -package rules rules/spec.yaml
```

---

@squat I've added the oapi-gen using `bingo get github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.9.0` and modified the Makefile too - but now I get this error on CI: 
```bash
/go/bin/jsonnet-v0.16.0 -m examples/manifests examples/main.jsonnet | xargs -I{} sh -c 'cat {} | /go/bin/gojsontoyaml-v0.0.0-20200602132005-3697ded27e8c > {}.yaml && rm -f {}' -- {
/go/bin/oapi-codegen-v1.9.0 -generate types,client,chi-server -package rules -o rules/rules.go rules/spec.yaml
bash: line 1: /go/bin/oapi-codegen-v1.9.0: No such file or directory
make: *** [Makefile:207: generate-rules-code] Error 127

Exited with code exit status 2
```
I think I missed some step but not sure yet..do you have any ideas?